src/components/Buttons/ToProductsPageButton.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { ToProductsPageButton } from "./ToProductsPageButton";
 import { Component } from "react";
 
 const meta = {
-    title: 'FestivalBooth/Buttons',
+    title: 'FestivalBooth/Buttons/ToProductsPageButtn',
```

Comment:
```suggestion
    title: 'FestivalBooth/Buttons/ToProductsPageButton',
```
スペルミス。ちなみに、Storybookはパスを自分で決める分、タイプミスするとディレクトリが分かれるよ。